Tasks/MavenV3/maventask.ts

```diff
@@ -193,8 +193,11 @@ async function execBuild() {
                         for (let i = 0; i < options.length; ++i) {
                             if ((options[i] === '--settings' || options[i] === '-s') && (i + 1) < options.length) {
                                 i++; // increment to the file name
-                                let suppliedSettingsXml: string = options[i];
-                                tl.cp(path.resolve(tl.cwd(), suppliedSettingsXml), settingsXmlFile, '-f');
+                                let suppliedSettingsXml: string = path.resolve(tl.cwd(), options[i]);
+                                // Avoid copying settings file to itself
+                                if (path.relative(suppliedSettingsXml, settingsXmlFile) !== '') {
```

Comment:
Could you please write some message for the case when destination and source paths are equal?